bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+
+### What is the scope of the BIPs repository?
+
+The BIPs repository is focused on information and technologies that aim to support and expand the utility of the bitcoin
+currency. Related topics that are of interest to the bitcoin community may be acceptable. Proposals that undermine the
+fundamental principle that Bitcoin constitutes a peer-to-peer electronic cash system for the bitcoin currency are
+out-of-scope.
+
+## BIP format and structure
+
+### Specification
+
+BIPs should be written in mediawiki or markdown[^markdown] format.
+
+Each BIP must have a _Preamble_, an _Abstract_, a _Copyright_, and a _Backward Compatibility_ section. Authors should
+consider all issues in the following list and address each as appropriate. For some BIPs, some concerns may not warrant
+a dedicated section.
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any bitcoin project to create an interoperable implementation.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe alternate designs that were considered and related work. The rationale
+  should address important objections or concerns raised.
+* Backward compatibility -- BIPs must include a section describing any backwards incompatibilities and their severity.
+  The BIP must explain how implementers/users should deal with these incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors, and documentation must be
+  completed before the BIP can be given the status "Proposed". Any or all of these can be provided either in the BIP or as
+  auxiliary files.
+* Changelog -- A section to track modifications to a BIP after reaching Proposed status.
+* Copyright -- The BIP must be placed under an acceptable license ([see BIP Licensing below](#bip-licensing)).
+
+#### BIP header preamble
+
+Each BIP must begin with an RFC 822-style header preamble. The headers must appear in the following order. Headers
+marked with "\*" are optional. All other headers are required. The overview is followed by an explanation for each header.
+
+```
+  BIP: <BIP number, or "?" before being assigned>
+* Layer: <Consensus (soft fork) | Consensus (hard fork) | Peer Services | API/RPC | Applications>
+  Title: <BIP title, up to 50 characters>
+  Author: <list of authors’ names and email addresses>
+  Status: <Draft | Proposed | Active | Abandoned>
```

Comment:
ACK. With respect to the discussion about the different statuses and their naming (and at the risk of glossing over the preceding detailed discussions), ISTM that if the main goal is to simplify and reduce them, then the simplest change and easiest one to switch over to would be what you have here: keep the Draft, Proposed, and Active statuses as-is (without renaming them), and drop the five others in BIP2 (Deferred/Rejected/Withdrawn/Final/Replaced/Obsolete) in favor of Abandoned, provided the finer-grained precision of those five has not added sufficient value to keep them.